index.html

```diff
@@ -2123,13 +2122,13 @@ <h3>Subject</h3>
         </p>
       </section>
       <section>
-        <h3>Signatures / Proofs</h3>
+        <h3>Proofs (aka Signatures)</h3>
```

Comment:
Not quite accurate... Signatures are one type of proof. Suggest using "e.g." instead?

Reply:
ok, look for updates to both by EOD

Reply:
@msporny should be done and ready to pull in.